### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by zbahati**
- Create INDEX of animal_id for visits
- Create INDEX of email in owners
- Create INDEX of vet_id for visits

# Finishes 

# Here is the screen shot for the queries results

**Before :**
peer-programmin1_720
peer-programmin2_720
peer-programmin3_720
peer-programmin4_720

**After:**
index_1_720
image
index_3

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @zbahati  & @MargaretTechworld  👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Please keep in mind that this is a pair programming activity, which means that each team member needs to make a substantial code contribution to the project. I noticed @zbahati didn't add any code to the PR (editing the readme doesn't count as a valid contribution) so please make sure each team member has a fair contribution. This will help ensure that the project is completed successfully and that everyone has the opportunity to contribute their skills and expertise. Thank you!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: schema.sql
Comment (AbelG101) [inline schema.sql:1]: - [ ] Even though indexes have been created for the `SELECT * FROM visits where vet_id = 2;` query the execution time in the screenshots AFTER the improvements were introduced is not significantly lower than 1 sec (1000 ms) so please make sure to work on this as it should take lower than 1sec after the improvements.
- Before:
  
- After:
  270368698-c65f6d47-5bf0-4586-a443-31581b3ad675
- Please make sure to update the screenshots accordingly after the update.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @zbahati & @MargaretTechworld  👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of `EXPLAIN ANALYZE` ✔
- Proper table indexing ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a good screenshot summary ✔


## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql
Comment (MargaretTechworld) [reply schema.sql:1]: Hi @AbelG101, 
Thanks for the review. The necessary correction has been made and the screen shots has been updated.
image

**Review by IngDavidTM — APPROVED** [review]
Hi @zbahati 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the Gitflow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**